packages/react-devtools-core/webpack.backend.js

```diff
@@ -46,6 +46,9 @@ module.exports = {
     libraryTarget: 'umd',
   },
   resolve: {
+    fallback: {
+      fs: false,
```

Comment:
Is it needed? By documentation Webpack 5 no longer polyfills Node.js core modules automatically. So it `false` by default.

Reply:
Can try removing it and we shall see.

Reply:
I think the `fs` "pollyfills" in Webpack configs were related to testing the named hooks feature, FWIW.

I think this `fallback` block can safely be removed from this config, like it was from the others. 👍🏼